extensions/quick-access/package.json

```
@@ -0,0 +1,125 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "quick-access",
+  "title": "Quick Access",
+  "description": "Quick copy, open, reveal the latest files in pinned Finder directories.",
+  "icon": "extension-icon.png",
+  "author": "koinzhang",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "pin-directory",
+      "title": "Pin Directory",
+      "icon": "pin-directory.png",
+      "subtitle": "Quick Access",
+      "description": "Pin directories that can be accessed quickly.",
+      "mode": "no-view"
+    },
+    {
+      "name": "access-directory",
+      "title": "Access Directory",
```

I think this describes the command better.

```suggestion
      "title": "Searched Pinned Directory",
```

---

> Could you add some screenshots? You can find more info [here](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots).
 Can you swap your light/dark icons? Right now you have dark background in the dark theme. It should be light background on the dark theme to have higher contrast.
 
I forgot the metadata, I will add it.
As for the icon issue, I will try to modify it, and if it doesn't perform well after swapping icon themes, I might just keep the light-colored icons

> I think this describes the command better.

Good idea, I will modify it。